test/common/http/filter_manager_test.cc

```diff
@@ -153,13 +155,14 @@ TEST_F(FilterManagerTest, SendLocalReplyDuringDecodingGrpcClassiciation) {
   EXPECT_CALL(filter_factory_, createFilterChain(_))
       .WillRepeatedly(Invoke([&](FilterChainManager& manager) -> bool {
         auto factory = createDecoderFilterFactoryCb(filter);
-        manager.applyFilterFactoryCb({}, factory);
+        manager.applyFilterFactoryCb({"filter1", "filter1"}, factory);
```

Comment:
Might be a good idea to name config and filter differently throughout, not obvious what this struct is.